src/app/util/ember-compatibility-functions.cpp

```diff
@@ -867,6 +867,7 @@ CHIP_ERROR WriteSingleClusterData(const SubjectDescriptor & aSubjectDescriptor,
 
         if (valueDecoder.TriedDecode())
         {
+            MatterReportingAttributeChangeCallback(aPath);
```

Comment:
Could you file a issue about fixing this bit of logic later when we do have data version support, to correctly only make this call if the data version ticked up?

Reply:
https://github.com/project-chip/connectedhomeip/issues/13013